Update DNS explained section (#42429)
mdn/content: files/en-us/learn_web_development/getting_started/web_standards/how_the_web_works/index.md

Real web addresses ([URLs](#components_of_a_url)) aren't the nice, memorable strings you type into your address bar to find your favorite websites. They are special numbers that look like this: `192.0.2.172`.

> This is called an {{Glossary("IP Address", "IP address")}}, and it represents a unique location on the web. However, it's not very easy to remember, is it? That's why the Domain Name System was invented. This system uses special servers that match up a web address you type into your browser (like `mozilla.org`) to the website's real (IP) address, which may vary depending on where you are in the world. Large websites are commonly made available on multiple servers, so that they load efficiently for different users worldwide.

Reviewer: A bit too much "world". Let's do this

```suggestion
This is called an {{Glossary("IP Address", "IP address")}}, and it represents a unique location on the web. However, it's not very easy to remember, is it? That's why the Domain Name System was invented. This system uses special servers that match up a web address you type into your browser (like `mozilla.org`) to the website's real (IP) address. Large websites are commonly made available on multiple servers, so that they load efficiently for different users worldwide. As a result, the IP address may vary depending on where you are. 
```
